OpenSim/Common/Test/testComponentInterface.cpp

```diff
@@ -830,6 +830,8 @@ void testListInputs() {
 
     cout << "  TableReporterMixedOutputs (contents)" << endl;
     cout << tabReporter->getReport() << endl;
+
+    tabReporter->clearReport();
```

Comment:
Could you add a simple one-line check after this to make sure the table is now empty?

```cpp
SimTK_TEST(tabReporter->getReport().getNumRows() == 0);
```